packages/components/src/color-picker/style.scss

```diff
@@ -126,7 +126,7 @@
 		0 0 1px 2px rgba(0, 0, 0, 0.4);
 	border-radius: 50%;
 	background-color: transparent;
-	transform: translate(-4px, -4px);
+	transform: translate(-7px, -7px);
```

Comment:
Would `translate(-50%, -50%)` work to prevent us from needing to update this in the future if the size changes again?

Reply:
Oh, YES!  That is definitely the best option for this.  Pushing the update now 😃